content/monitors/notifications.md

```diff
@@ -27,98 +27,139 @@ right people get notified so the problem can be resolved as soon as possible.
 
-Here is an example of how you can use template variables for a multi alert:
+These variables use simple if-else logic to display a different message depending on whether the event is a trigger, warning, recovery, or no data notification with the following syntax:
 
-{{< img src="monitors/notifications/templatevareditor.png" alt="template var editor" responsive="true" popup="true" >}}
+{{< img src="monitors/notifications/conditionalvars.png" alt="conditional vars" responsive="true" popup="true" >}}
 
-and the corresponding event notification:
+Here is an example of how you can set it up in the editor:
 
-{{< img src="monitors/notifications/templatevar.png" alt="template var" responsive="true" popup="true" >}}
+{{< img src="monitors/notifications/templateconditionaleditor.png" alt="template conditional editor" responsive="true" popup="true" >}}
 
-The tag template variables available depend on the tag group selected in Step 1 of the monitor editor. The possible options automatically populate at the bottom of the "Use message template variables" help box in Step 3 of the editor.
-These variables can also be used in the monitor titles (names), but note that the variables are only populated in the text of Datadog child events (not the parent, which displays an aggregation summary).
+The corresponding trigger event notification looks like this:
 
-Some tags identifying your triggering scope are automatically inserted into the title of your multi alert. If your scope is defined by a lot of tags, your alert title may end up being undesirably long. If you've used template tag variables to include this information in the body of your alert, you can uncheck
+{{< img src="monitors/notifications/templateconditionaltrigger.png" alt="template conditional trigger" responsive="true" popup="true">}}
 
-### Include triggering tags in notification title
+and the recovery notification:
 
-In order to save some space, you can Include triggering tags in notification title. This makes your notification title look like this:
+{{< img src="monitors/notifications/templateconditionalrecover.png" alt="template conditional recover" responsive="true" popup="true" >}}
 
-{{< img src="monitors/notifications/templatevar_short.png" alt="template var short" responsive="true" popup="true" >}}
+#### `{{is_recovery}}` or `{{is_alert_recovery}}` 
 
-Note that template variable content is escaped by default. If your variable
-contains JSON or code that you would NOT like to be escaped, then use triple braces instead of double braces (e.g. `{{{event.text}}}`).
+* `{{is_recovery}}` triggers and a monitor recovers indifferently either from a **WARNING** state or an **ALERT** state. 
+* `{{is_alert_recovery}}` triggers when a monitor recovers directly from an **ALERT** state to **OK** state. 
+* `{{is_warning_recovery}}` triggers when a monitor recovers from a **WARNING** state to **OK** state
+
+This means that if the monitor switches from ALERT to WARNING then OK state:
```

Comment:
Could you please reword like this?

```
This means that if the monitor switches from an **ALERT** to a **WARNING** to an **OK** state:
```

Reply:
I also bolded the state names because they were bolded in the previous section.

Reply:
well done 👌